tests/unit_and_functional/test_cli_service_deployments.py

```diff
@@ -24,7 +24,7 @@
 
 from bodywork.cli.service_deployments import (
     delete_service_deployment_in_namespace,
-    display_service_deployments_in_namespace,
+    display_service_deployments,
```

Comment:
Do we need to add a test for the no namespace condition?

Reply:
Yes we should because it is effectively a new use case i.e. Display all service deployments, even though we do test this in `k8s.service_deployments`.